doc/developers/contributing.rst

```diff
@@ -181,6 +192,16 @@ Contributing code
   so far. In order to claim an issue for yourself, please comment exactly
   ``/take`` on it for the CI to automatically assign the issue to you.
 
+To maintain the quality of the codebase and ease the review process, any
+contribution, including code generated by language models (LLMs), must conform
+to the project's existing style guide and coding conventions, in particular:
+
+- Keep all lines under 88 characters.
```

Comment:
same here. it's linter's job

Reply:
What I had in mind is to avoid contributions that would just result in a red CI. In the case of https://github.com/scikit-learn/scikit-learn/pull/29268#pullrequestreview-2122246704, I would have liked to refer the OP to an official statement in the contributor's guide regarding raw copy-paste from an LLM.

That being said, it's true that the bot already comments on such CI fails, but I have the feeling that raw-LLM contributions are going to become more and more of a problem. WDYT?

Reply:
I don't think so, at least for now we haven't had many. And when they get them, we simply block them in most cases anyway, so we don't care about the linter issues.

Reply:
And there is no reason to believe that LLMs wouldn't be better at following format standard than the average human.

The real problem with LLM code contributions, is not necessarily their form or ability to make tests pass but the fact that no human contributor fully understands the goal of the contribution, its inner value for end-users and more generally if this a change of the code base that is useful for our community or not.